### PR TITLE
Prepare for the team migration

### PR DESCRIPTION
**Pull request by apidesigner**
Update README to follow adidas GH guidelines.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,3 +41,19 @@ Refer to [RFC2119](https://www.ietf.org/rfc/rfc2119) for details.
...
+ 
+© adidas AG
+
+adidas AG publishes this software and accompanied documentation (if any) subject to the terms of the MIT license with the aim of helping the community with our tools and libraries which we think can be also useful for other people. You will find a copy of the MIT license in the root folder of this package. All rights not explicitly granted to you under the MIT license remain the sole and exclusive property of adidas AG.
+ 
+NOTICE: The software has been designed solely for the purpose of providing API design and development guidelines. The software is NOT designed, tested or verified for productive use whatsoever, nor or for any use related to high risk environments, such as health care, highly or fully autonomous driving, power plants, or other critical infrastructures or services.
+ 
+If you want to contact adidas regarding the software, you can mail us at _software.engineering@adidas.com_.
+ 
+For further information open the [adidas terms and conditions](https://github.com/adidas/adidas-contribution-guidelines/wiki/Terms-and-conditions) page.
```
Comment (moelders) [inline README.md]: Add the link to the license too:

```
### License

[MIT](LICENSE)
```

Comment (moelders) [inline README.md]: It is a good idea to have the `CONTRIBUTING.md` and the `CODE_OF_CONDUCT.md` files.

You can see an example:

- [`CONTRIBUTING.md`](https://github.com/adidas/js-linter-configs/blob/master/.github/CONTRIBUTING.md).
- [`CODE_OF_CONDUCT.md`](https://github.com/adidas/js-linter-configs/blob/master/CODE_OF_CONDUCT.md).

Comment (moelders) [inline README.md]: In the `README.md` it is not necessary to have this header:

```
# Introduction

![adidas logo](https://adidas-group.gitbooks.io/api-guidelines/content/assets/adidas-logo.svg)

## adidas API Guidelines

_Guidelines for API...
```

You can just do this (and remove one `#` character from the sub points with `###`):

```
![adidas logo](https://adidas-group.gitbooks.io/api-guidelines/content/assets/adidas-logo.svg)

# adidas API Guidelines

_Guidelines for API...
```

Comment (apidesigner) [reply README.md]: Done.

Comment (apidesigner) [reply README.md]: Is this needed? If so please add it to the confluence required steps. If not, please let's move forward.

Comment (apidesigner) [reply README.md]: It is actually necessary for GitBook rendering (https://adidas.gitbook.io/api-guidelines/). The README file is used to render the front page of the API Guidelines.

Comment (apidesigner) [reply README.md]: Actually I need to align this with the rendering at GitBook which can be only done when this branch is merged. 

This very README is rendered at https://adidas.gitbook.io/api-guidelines/ and as such the heading and levels actually matters. I will. however reorganize the headings once this PR is merged.

Comment (moelders) [reply README.md]: It is not really mandatory, but if you want someone else to contribute to the repository is a good idea to show the contribution guidelines.

All this useful information is in the _Good practices for public code_ section.

Comment (moelders) [reply README.md]: OK, I understand.

Comment (apidesigner) [reply README.md]: Added.

**Comment by apidesigner** [comment]
@moelders changes pushed, please revisit the review, so we can merge it and move forward.

**Pull request merged**